### ISSUE TITLE
WCAG 2.2 Violation - There is a custom combobox/listbox component present within the interface that is missing role and state information.

### ISSUE THREAD

**Issue by ivrouwe-ubc** [ubc/wp-comments-chat-ui#16]

**Comment by ivrouwe-ubc** [comment]
Closing this Issue, as I've determined that the whole custom combobox/listbox present in the interface will need an overhaul (instead of us remediating discrete issues).
